### PR TITLE
security page

### PR DESCRIPTION
**Pull request by jospoortvliet**
Created a new security page as this is one of our major strengths.
Let me know what you think.
@LukasReschke more details are welcome
@jancborchardt @Espina2  feedback on looks welcome too.,

secure nextcloud

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@Espina2 feel free to simply go in and make changes to the design/layout, of course.

**Comment by Espina2** [comment]
Just have a quick look, and I think we wave many information at the same time. We should have more white space and better alignments. I will make this page my priority. :)

File: page-secure.php [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+			<p>Learn how to use server side encryption in our <a href="<?php echo $DOCUMENTATION_ADMIN; ?>configuration_files/encryption_configuration.html" target="_blank" rel="tooltip" title="Server Side Encryption Configuration">documentation</a></p>
+		</div>
+		<div class="col-md-5">
+			<img class="img-responsive screenshot" src="<?php echo get_template_directory_uri(); ?>/assets/img/features/encryption-recovery.png" alt="in action" />
+		</div>
+	</div>
+</div>
+
+
+<div class="wrap container featureblock">
```
Comment (LukasReschke) [inline page-secure.php]: Is this element ever closed?

File: assets/css/secure.scss [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+	}
+}
+
+/* Process image */
+
+.process {
+	height: 1100px;	
+
+	h1 {
+      font-size: 25px;
```
Comment (jancborchardt) [inline assets/css/secure.scss]: Just a detail but the indentation seems to be all mangled in these files. We use tabs everywhere. ;)

File: page-secure.php [hunk 3]
```diff
@@ -0,0 +1,209 @@
+<div class="container-fluid background">
+	<div class="container">
+		<div class="col-md-6 topheader">
+			<h1>Security and Authentication</h1>
```
Comment (jancborchardt) [inline page-secure.php]: »authentication« lowercase here? Because we only capitalize the first word of headers.

File: page-secure.php [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+			<h2>Nextcloud is designed to protect user data through multiple layers of protection.</h2>
```
Comment (jancborchardt) [inline page-secure.php]: > Nextcloud is designed to protect user data
> through multiple layers of protection

Add a <br> between »data« and »through« so it doesn’t weirdly break. Also, no dot at the end.

File: page-secure.php [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+		</div>
+	</div>
+</div>
+
+
+<div class="container featureblock">
+	<div class="row">
+		<div class="col-md-7">
+			<h2>Authentication capabilities</h2>
+			<p>The Nextcloud login system supports pluggable authentication including two-factor authentication and device specific passwords, complete with a list of connected browsers and devices on the users’ personal page.</p>
```
Comment (jancborchardt) [inline page-secure.php]: »log in« are two words. ;)

**Comment by jancborchardt** [comment]
Just some wording nitpicks. I’m sure @Espina2 can come up with an awesome layout. :) We should remove the border-radius from any images as that looks odd.

Also – we need a page like this for Design! :) cc @nextcloud/designers ref https://github.com/nextcloud/nextcloud.com/pull/140

**Comment by skjnldsv** [comment]
Don't have time to look into details, right now, but it looks pretty amazing!! :scream:

**Comment by jospoortvliet** [comment]
guys, while this page might not be perfect, it is certainly as good as the features page for example - how about we put this live? It's a nice step forward, which is good enough to me ;-)

Waiting longer to merge just increases the chance for merge conflicts...

**Comment by jospoortvliet** [comment]
Esp @jancborchardt what do you think? I made the changes you requested, btw.

File: page-secure.php (hunk 1 above)
Comment (jospoortvliet) [inline page-secure.php]: whoops, fixed.

File: assets/css/secure.scss (hunk 2 above)
Comment (jospoortvliet) [inline assets/css/secure.scss]: yeah, I know, it's a mess.... Should run a tool at some point which fixes it or something. Sorry!

**Comment by LukasReschke** [comment]
@jospoortvliet Can you deploy this branch to staging?

**Comment by jospoortvliet** [comment]
* [ ] make it more clear that the bar is click-able
* [ ] shorten the bar one item?!?!
* [ ] add links to documentation about hardening etc etc

**Comment by Henni** [comment]
Looks great!
Only the `processimg` seems to need a bit more work:
* [ ] The graphic splits apart on smaller screens. I'd prefer it if it would keep it's style similar to how it looks on larger screens.

image
* [ ] The first link has a negative left margin of `-25px` which misaligns the image.

image

Both of these points are only small issues and might be fixed in a separate PR.

**Comment by jancborchardt** [comment]
- [ ] can we have the arrow graphics as SVG? They look blurry on retina and when zoomed.
- [ ] there are multiple h1 on the page which creates visual strangeness. Any page should have only one h1 (»Security and authentication« in this case) the rest is h2’s or h3’s respectively. (»Security process« should be h2, Requirements h3, Security bug bounties and Auth capabilities h2, Screenshots h2 again)
- [ ] the HackerOne logo is too large and should be above the image instead of above the header

**Comment by Espina2** [comment]
I created this illustration and I pretend to animate them. I think that communicate better what we are talking about. I already discuss this with @jospoortvliet and It can be done in a 2.0 version of this page.

What do you think @jancborchardt 

screen shot 2016-10-14 at 14 23 53

**Comment by jancborchardt** [comment]
@Espina2 you rock! :metal: Fits the header style better indeed. :)

**Comment by jospoortvliet** [comment]
Done all of it. Check it out ;-)
https://nextcloud.com/secure/?staging=true

Ok to merge? We can add the nice image from Paulo once he has it finished in a separate PR.

**Comment by jancborchardt** [comment]
Just 1 thing: Change the h2 styles from 60px (same as h1) to 40px. And h3 from 35px to 30px. Then it’s ok. :)

**Comment by skjnldsv** [comment]
capture d ecran_2016-10-18_08-57-11

padding incorrect for 2 & 3 :)

**Comment by jancborchardt** [comment]
@skjnldsv and why is the font serif on your machine? ;)

**Comment by skjnldsv** [comment]
Because you're using Open Sans on the svg and I don't have it installed :p

**Comment by skjnldsv** [comment]
ps: 
```
$ svgo Desktop/requirements.svg 
Done in 127 ms!
1.977 KiB - 65.8% = 0.677 KiB
```

**Comment by jospoortvliet** [comment]
@skjnldsv what padding is wrong?

**Comment by jospoortvliet** [comment]
All changes are deployed, btw:
https://nextcloud.com/secure/?staging=true

**Comment by skjnldsv** [comment]
@jospoortvliet 👍 for svg optimisation! :)
I meant the second and third. The text seems too far on the left!

**Comment by jospoortvliet** [comment]
@skjnldsv yeah, the text is center aligned. It probably would look better to actually not do that but put the longer texts a bit more to the right... but I'm no designer so I just do 'center center' and done. Will deploy now, we can fix this, still, though, if you and perhaps Jan/Paulo agree it should be aligned differently.

**Comment by skjnldsv** [comment]
👍 
Still some svg to do but great! :)

```
./assets/files/Nextcloud-Box-Assembly-Guide.svg

Done in 1047 ms!
263.274 KiB - 61% = 102.566 KiB

./assets/img/enterprise/education.svg

Done in 98 ms!
3.998 KiB - 57.5% = 1.7 KiB

./assets/img/enterprise/nocheck.svg

Done in 116 ms!
4.403 KiB - 97.8% = 0.096 KiB

./assets/img/enterprise/check.svg

Done in 124 ms!
4.095 KiB - 75.1% = 1.02 KiB

./assets/img/enterprise/optional.svg

Done in 127 ms!
4.425 KiB - 78.3% = 0.959 KiB

./assets/img/enterprise/publicsector.svg

Done in 116 ms!
9.551 KiB - 61.9% = 3.644 KiB

Done in 178 ms!
12.438 KiB - 18.7% = 10.112 KiB

./assets/img/conference/conf-logo.svg

Done in 158 ms!
23.613 KiB - 45.2% = 12.945 KiB

./assets/img/5years/5years.svg

Done in 118 ms!
8.124 KiB - 8.5% = 7.435 KiB

./assets/img/social/rss.svg

Done in 68 ms!
0.725 KiB - 17% = 0.602 KiB

./assets/img/social/twitter.svg

Done in 81 ms!
1.473 KiB - 2.3% = 1.439 KiB

./assets/img/federation/federation-logo.svg

Done in 116 ms!
5.12 KiB - 10.7% = 4.572 KiB

./assets/img/history/office.svg

Done in 106 ms!
3.428 KiB - 56.7% = 1.485 KiB

./assets/img/history/release.svg

Done in 128 ms!
6.576 KiB - 30.6% = 4.562 KiB

./assets/img/history/desktop.svg

Done in 105 ms!
3.458 KiB - 51.7% = 1.67 KiB

./assets/img/history/org.svg

Done in 137 ms!
6.576 KiB - 30.6% = 4.562 KiB

./assets/img/history/github.svg

Done in 106 ms!
3.42 KiB - 52.3% = 1.633 KiB

./assets/img/history/meeting.svg

Done in 107 ms!
3.605 KiB - 49.4% = 1.825 KiB

./assets/img/history/neutral.svg

Done in 149 ms!
14.93 KiB - 16.7% = 12.443 KiB

./assets/img/history/inc.svg

Done in 165 ms!
14.88 KiB - 16.3% = 12.46 KiB

./assets/img/history/mobile.svg

Done in 106 ms!
3.351 KiB - 53% = 1.573 KiB

./assets/img/history/code.svg

Done in 106 ms!
3.403 KiB - 52.5% = 1.618 KiB

./assets/img/history/press.svg

Done in 106 ms!
3.332 KiB - 53.2% = 1.559 KiB

./assets/img/history/money.svg

Done in 107 ms!
3.531 KiB - 50.7% = 1.739 KiB

./assets/img/history/statistics.svg

Done in 106 ms!
3.19 KiB - 55.8% = 1.409 KiB
```

**Pull request merged**